src/components/layouts/LoginPage.tsx

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable @typescript-eslint/no-misused-promises */
+// LoginPage.jsx
+import { useState } from 'react';
+import { Button, Card, Form, Input, message } from 'antd';
+import { useNavigate } from 'react-router-dom';
+
+const LoginPage = () => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [loading, setLoading] = useState(false);
+  const navigate = useNavigate();
+
+  const onFinish = async (values: any) => {
+    try {
+      const { usernmae, password } = values;
+      setLoading(true);
+      const response = await fetch(
+        `${process.env.REACT_APP_API_URL}/user/login`,
```

Comment:
Please use `API_BASE_URL` instead of `process.env.REACT_APP_API_URL}` and don't use hard coded api endpoint like this `/user/login` instead use it dynamically from `endpoints.ts`